modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/paths/CP2TeamMembers.path

```diff
@@ -25,6 +25,16 @@
 	<td>//select[contains(@name, 'invites[${key_index}]')]//option[contains(text(), '${role}')][@disabled]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>ROLE_OPTIONS</td>
+	<td>//option[contains(text(), '${key_option}')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>DROPDOWN</td>
+	<td>//select[contains(@class, 'form-control')]</td>
+	<td></td>
+</tr>
```

Comment:
Did you run source-format for this, Luana? I think it will fail since it is not sorted alphabetically.

Reply:
I ran. I will fix it in the hand.